docs/dev/git-cheat.md

```diff
@@ -0,0 +1,126 @@
+# Git cheat sheet for (busy) contributors
+
+## Requirement: DCO and signing commits
```

Comment:
"DCO" needs explanation

Reply:
It should be also clarified that there are two kinds of "signing": DCO and GPG(/SSH)